workflow/usage.rst

```diff
@@ -177,10 +177,23 @@ what actions are allowed on a blog post::
 Using Events
 ------------
 
-To make your workflows even more powerful you could construct the ``Workflow``
+To make your workflows more flexible, you can construct the ``Workflow``
 object with an ``EventDispatcher``. You can now create event listeners to
-block transitions (i.e. depending on the data in the blog post). The following
-events are dispatched:
+block transitions (i.e. depending on the data in the blog post) and do
+additional actions when a workflow operation happened (i.e. sending 
+announcements).
+
+Each step has three events that are fired in order:
+
+- An event for every workflow;
+- An event for the workflow concerned;
+- An event for the workflow concerned with the specific transition or place name;
+
+The following events are dispatched:
+
+* ``workflow.guard``
+* ``workflow.[workflow name].guard``
+* ``workflow.[workflow name].guard.[transition name]``
```

Comment:
i found it illogic that these events are shown below, when they are also events, just a special type of events.